mlflow/server/js/src/experiment-tracking/routes.js

```diff
@@ -3,8 +3,12 @@ import { X_AXIS_RELATIVE } from './components/MetricsPlotControls';
 class Routes {
   static rootRoute = '/';
 
-  static getExperimentPageRoute(experimentId) {
-    return `/experiments/${experimentId}`;
+  static getExperimentPageRoute(experimentId, isComparingRuns = false) {
+    if (isComparingRuns) {
+      return `/experiments/${experimentId}?isComparingRuns=true`;
+    } else {
+      return `/experiments/${experimentId}`;
+    }
```

Comment:
Update "Give it a try" link to point to the chart view, rather than the table view (see https://github.com/mlflow/mlflow/pull/7864/files#r1111338871)